evm/deploy-its.js

```diff
@@ -97,7 +100,7 @@ async function deployImplementation(config, wallet, chain, options) {
                     deployMethod,
                     wallet,
                     getContractJSON('InterchainToken'),
```

Comment:
pass artifactPath for other calls as well?

Reply:
yeah